TESTING_GUIDE.md

```diff
@@ -29,6 +29,18 @@ We have devised our own test framework to test the spec against an OpenSearch cl
 
 ### Running Spec Tests Locally
 
+Prerequisites: 
+Download and install latest version of Node.js and npm from 
+[https://docs.npmjs.com/downloading-and-installing-node-js-and-npm]
+```bash
+Note: to download the latest version of npm, on the command line, run the following command:
```

Comment:
Remove this note. If you don't have `npm` you cannot run `npm -g`. Running it again with `npm -g npm` is not useful either. It's not our business here to explain how to upgrade npm.